modnet/hyper_opt/fit_genetic.py

```diff
@@ -33,25 +34,35 @@ def __init__(
             weights (Dict[str, float]): Optional (for joint learning only). The relative loss weights to apply for each target.
         """
 
+        self.act = "elu"
+        self.loss = "mae"
```

Comment:
Would this make it deterministic though? Since the models will in any case lead to somewhat slightly different results each time they are fitted, they might differ from iteration to iteration?